plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -723,12 +741,15 @@ public void testUpdateAllValues()
         });
     }
 
+    @Test
     @Override
     public void testWrittenStats()
     {
         // TODO Kudu connector supports CTAS and inserts, but the test would fail
     }
 
+    @Test
+    @Timeout(value = 180, unit = SECONDS)
```

Comment:
is this "inlined" from super?

Reply:
yes

Reply:
does this mean there can be child tests that don't have any timeouts even if super specified one?

(e.g. I'm thinking about DF tests which can actually time-out).

No change requested.